bin/disco_vpc_ui.py

```diff
@@ -28,6 +28,9 @@ def parse_arguments():
                                help='What to call the new environment.')
     parser_create.add_argument('--type', dest='vpc_type', required=True,
                                help='What type of environment to create (as defined in config).')
+    parser_create.add_argument('--skip-enis', dest='skip_enis', action='store_const',
```

Comment:
I was just thinking in the case of creating sandboxes, not all hostclasses with static ENIs need to be spun up. That's why I added the option here. At the same time, there's not much cost to allocate these ENIs either, so I am ok either way.